assets/airports/lipz.json

```diff
@@ -0,0 +1,2300 @@
+            "body": [["INDOD", "A50+"], ["ERSAB", "A50+"]],
+            "rwy": {
+                "LIPZ22L": [],
+                "LIPZ22R": []
+            },
+            "draw": [
+                ["CHI", "INDOD", "ERSAB"]
+            ]
+        },
+        "INDOD1W": {
+            "icao": "INDOD1W",
+            "name": "Indod One Whiskey",
+            "entryPoints": {"INDOD": [["INDOD", "A40+"]]},
+            "body": [["ERSAB", "A40+"]],
+            "rwy": {
+                "LIPZ22L": [],
+                "LIPZ22R": []
+            },
+            "draw": [
+                ["INDOD", "ERSAB"]
+            ]
+        },
+        "OTGIG1E": {
+            "icao": "OTGIG1E",
+            "name": "Otgig One Echo",
+            "entryPoints": {"OTGIG": [["OTGIG", "A90+|A110-|S270-"]]},
+            "body": [["PZ655", "A60+|S230-"], ["PZ850", "A60+|A90-|S230-"], ["PZ845", "A60+|A90-|S230-"], ["PZ840", "A60+|A90-|S210-"], ["PZ735", "A50+|S210-"], ["PZ730", "A50+|S210-"], ["PZ725", "A40+|S210-"], ["LAREN", "A40+|S210"]],
+            "rwy": {
+                "LIPZ04L": [],
+                "LIPZ04R": []
+            },
+            "draw": [
+                ["OTGIG", "PZ655", "PZ850", "PZ845", "PZ840", "PZ735", "PZ730", "PZ725", "LAREN"]
+            ]
+        },
+        "OTGIG1F": {
+            "icao": "OTGIG1F",
+            "name": "Otgig One Foxtrot",
+            "entryPoints": {"OTGIG": [["OTGIG", "A90+|A110-|S270-"]]},
+            "body": [["ERSAB", "A50+|S210"]],
+            "rwy": {
+                "LIPZ22L": [],
+                "LIPZ22R": []
+            },
+            "draw": [
+                ["OTGIG", "ERSAB"]
+            ]
+        },
+        "OTGIG1P": {
+            "icao": "OTGIG1P",
+            "name": "Otgig One Papa",
+            "entryPoints": {"OTGIG": [["OTGIG", "A50+"]]},
+            "body": [["^_OTGIG331012", "A50+"], ["LAREN", "A40+"]],
+            "rwy": {
+                "LIPZ22L": [],
+                "LIPZ22R": []
+            },
+            "draw": [
+                ["OTGIG", "_OTGIG331012", "LAREN"]
+            ]
+        },
+        "OTGIG1R": {
+            "icao": "OTGIG1R",
+            "name": "Otgig One Romeo",
+            "entryPoints": {"OTGIG": [["OTGIG", "A50+"]]},
+            "body": [["CHI", "A50+"], ["INDOD", "A50+"], ["ERSAB", "A50+"]],
+            "rwy": {
+                "LIPZ22L": [],
+                "LIPZ22R": []
+            },
+            "draw": [
+                ["OTGIG", "CHI", "INDOD", "ERSAB"]
+            ]
+        }        
+    },
+    "spawnPatterns": [
+        {
+            "origin": "",
+            "destination": "LIPZ",
+            "category": "arrival",
+            "route": "RESIA.P131.ADOSA.L612.ALBET.ALBET1E.LIPZ04R",
+            "altitude": [6000, 8000],
+            "speed": 250,
+            "method": "random",
+            "rate": 10,
+            "airlines": [
+                ["aal/long", 6],
+                ["afr", 18],
```

Comment:
You can also use fleets to make sure airlines like Air France only fly smaller planes and no 777s